runtime/hydradx/src/xcm.rs

```diff
@@ -413,6 +413,7 @@ impl Contains<RuntimeCall> for SafeCallFilter {
 				| RuntimeCall::Currencies(..)
 				| RuntimeCall::Tokens(..)
 				| RuntimeCall::OrmlXcm(..)
+				| RuntimeCall::PolkadotXcm(pallet_xcm::Call::send { .. },)
```

Comment:
@green-jay i don't think we can unfilter it here, is shouldn't be necessary anyway

cc @Roznovjak

Reply:
btw. seems weird that OrmlXcm is unfiltered as well